### ISSUE TITLE
VAMC Police: How to request a Police report section - Review PR

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#20855]
## Description
Add How to request a Police report section to VAMC Police page

### Additional notes
There are existing accessibility and padding issues with this component that should be resolved during migration:
1. The heading is currently a semantic `<h2>` with a VADS class to style it as an H3. It should actually be a semantic `<h3>`. I looked back at the history and couldn't find a reason why this wouldn't have been done originally so I think it was just missed accidently.
   - Please change the heading "How to request a Police report" to a semantic `<h3>`
   - There should also be no VADS classes on this element, no class to change the font size and no classes for padding
   - Screenshot of the correct code:
![Image](https://github.com/user-attachments/assets/539420ad-dabe-459d-8929-20c85c6c2719) 
 
2. There are other VADS classes on a couple other elements that need to be removed (this will resolve [this existing Sitewide ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18103)):
   -  On the wrapping `<div>` inside the component, containing the text and link, all the classes here need to be removed:
![Image](https://github.com/user-attachments/assets/cd68571c-b506-4667-8a2c-62a33213c404)
   - Also, on the wrapping `<div>` that contains the component itself, the is a VADS class for margin, but this isn't doing anything. It is superfluous and to keep the code clean and make sure this doesn't impact anything in the future should be removed:
![Image](https://github.com/user-attachments/assets/0992b972-3d18-4758-b78d-24c24a9c9d05)

### Screenshots

**Current**

![Image](https://github.com/user-attachments/assets/4985f54b-dab8-437d-a25d-a70dc0318e03)

**With requested updates**

![Image](https://github.com/user-attachments/assets/690de94d-7fe1-44c2-93a0-1a1402bd285f)

### Acceptance Criteria
- [x] Add How to request a Police report section to VAMC Police pages
- [x] Heading is a `<h3>`
- [x] Component is using the default padding and styles of the component, no extra VADS classes are added

**Comment by eselkin** [comment]
WIP on next-build for CCFeaturedContent / summary-box
<img width="1391" alt="Image" src="https://github.com/user-attachments/assets/aa47b741-87fb-48d1-aa42-54cd72e03b5c" />

**Comment by laflannery** [comment]
To Do: 
- Review the existing PR: https://github.com/department-of-veterans-affairs/next-build/pull/961
- Get it update to date so there are no more conflicts
- Complete anything necessary and submit PR as usual for code review and QA

**Comment by omahane** [comment]
Per Laura, the report box should have a heading level of h3, which means that it no longer has its own entry in the "On this page" table of contents. This is expected and desired, as the report is another way to contact the police, so it should now be appropriately nested in the DOM.
